docker-entrypoint.sh

```diff
@@ -9,6 +9,17 @@ if [ "$MODE" != "disabled" ] && [ "$MODE" != "staging" ] && [ "$MODE" != "produc
     exit 1
 fi
 
+# $DOMAIN value can be a single domain or multiple domains separated by commas.
+# Preparing a $domain_main that will be used for the self-signed certificate and multi-domain Certbot command.
+c="${DOMAIN//[^,]}"
+dcount=$(( ${#c} + 1 ))
+domain_main=$(echo $DOMAIN | cut -d "," -f 1)
+certbot_domains_cmd=""
+for i in `seq 1 $dcount`; do
+    d=$(echo $DOMAIN | cut -d "," -f $i)
+    certbot_domains_cmd="$certbot_domains_cmd -d $d"
```

Comment:
uppercase variable name, and `DOMAINS_ARGS` would be more appropriate (it's not a command, only arguments)